docs/content/users/install/docker-installation.md

```diff
@@ -46,6 +45,27 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     !!!warning "Colima can only work in your home directory unless you do further configuration"
         By default, Colima only mounts your home directory, so it’s easiest to use it in a subdirectory there. See the `~/.colima/default/colima.yaml` for more information, or notes in [colima.yaml](https://github.com/abiosoft/colima/blob/fc948f8f055600986f87e29e3e632daf56ac8774/embedded/defaults/colima.yaml#L130-L143).
 
+    ### Lima
+
+    [Lima](https://github.com/lima-vm/lima) is a free and open-source project supported by the [Cloud Native Computing Foundation](https://cncf.io/).
+
+    1. Install Lima with `brew install lima`.
+    2. Install the `docker` client if you don't already have it with `brew install docker`.
+    3. Create a default Lima setup with 4 CPUs, 6GB memory, 100GB storage, and Cloudflare DNS, adjusting as needed:
+    ```
+    limactl create --vm-type=vz --mount-type=virtiofs --mount-writable --memory=6 --cpus=4 --disk=100 template://docker
+    docker context create lima-default --docker "host=unix://$HOME/.lima/default/sock/docker.sock"
+    docker context use lima-default
+    ```
+    After the initial run above, you can use `limactl start`.  Run `limactl list` to see configured setup.
```

Comment:
```suggestion
    After the initial setup, you can start your instance with `limactl start`. Run `limactl --help` to see the list of available commands.
```
instead of adding the command to see the configured setup i think providing the help command to get the list of all available command might be more helpful (which also includes the list command)

Reply:
I like it as is. People have to branch out to learn more.